assets/app/view/game/player.rb

```diff
@@ -41,11 +42,16 @@ def render
       end
 
       def render_title
+        bg_color = if setting_for(:show_player_colors, @game)
+                     player_colors(@game.players)[@player] || color_for(:bg2)
```

Comment:
I don’t really care about it, but something like `convert_hex_to_rgba(route_prop(idx % ROUTE_COLORS.size, 'color'), 0.5)` for players 9+ in `player_colors` might work.
=> no merge, because I’m not sure if you want to invest more time into this.

Reply:
Nor me :)